src/v2/guide/components.md

```diff
@@ -190,7 +190,7 @@ data: {
 }
 ```
 
-À la place, **une option du composant `data` doit être une fonction**, afin que chaque instance puisse conserver une copie indépendante de l'objet retourné :
+À la place, **un attribut du composant `data` doit être une fonction**, afin que chaque instance puisse conserver une copie indépendante de l'objet retourné :
```

Comment:
Je ne suis pas d'accord avec cette appelation. L'attribut est un terme utilisé pour un attribut de balise HTML.

Ici on remplace l'objet initialement dans data par une function. Et habituellement ce que l'on nomme l'objet « options » est l'objet que nous passons à l'instance de Vue (dont l'une des *propriétés* (et pas *attribut*) est `data`. Je n'ai plus l'original sous la main puisque cela va être remplacé par la nouvelle version de cette page. Mais si cela n'est pas clair je préfèrerais plutôt : 

```
À la place, **la propriété `data` de l'objet d'option du composant doit être une fonction**
```
ce qui me semble plus juste ?

Reply:
À la ligne 183 de ce fichier le titre `### data Must Be a Function` n'est pas traduit. Tu peux le traduire par

```
`data` doit être une fonction
```

?

Reply:
D'ailleurs je ne suis pas du tout experte en traduction, je me débrouille en anglais mais de là à dire que je puisse traduire correctement des paragraphes... Du coup si je retombe sur des phrases non traduites au fil de ma lecture (et relecture par la même occasion), tu préfères que je tente une traduction ou que je laisse le passage en anglais tel quel ?